scripts/templates/print.hpp.mako

```diff
@@ -401,9 +409,13 @@ template <typename T> inline void serializePtr(std::ostream &os, const T *ptr) {
         os << *ptr;
         os << ")";
     }
+
+    return ${X}_RESULT_SUCCESS;
 }
 
-inline int serializeFunctionParams(std::ostream &os, uint32_t function, const void *params) {
+///////////////////////////////////////////////////////////////////////////////
+// @brief Print function parameters
+UR_DLLEXPORT inline ${x}_result_t UR_APICALL ${x}FunctionParamsPrint(std::ostream &os, uint32_t function, const void *params) {
```

Comment:
I'm not sure if this is useful outside of urtrace (but I guess someone might want to implement a similar functionality themselves).

```suggestion
UR_DLLEXPORT inline ${x}_result_t UR_APICALL ${x}FunctionParamsPrint(std::ostream &os, enum ur_function_t function, const void *params) {
```

Reply:
Done